src/components/app/App.tsx

```diff
@@ -19,15 +21,15 @@ class Props {
 export class Evergreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    // TODO: get the API client from some config
     this.state = {
-      APIClient: rest.EvergreenClient("admin", "e4f2c40463dcade5248d36434cb93bac", "http://localhost:8080/api"),
+      APIClient: rest.EvergreenClient("", "", "", ""),
```

Comment:
We should initialize this to be null rather than a blank client

Reply:
I originally had this as null in my code earlier today, but I noticed that if you initialize it to null and do not upload a config file first, the admin page will not load. It fails because you can't call ConvertToAdminSettings with a null API client, but you can with an empty one. Should I still change it for now?

Reply:
Strange, that shouldn't happen since the function doesn't use the client at all. Let's initialize it to null and fix it in evergreen.js if needed